### DISCUSSION TITLE
Combining P5js with strudel

### DISCUSSION THREAD

**Discussion by mgmg45** [tidalcycles/strudel#160] [General]
Has anyone attempted to combine P5Js with strudel??
I'm trying my best now, i think this would be the ultimate combo

So there is either AudioContext from P5 or AudioContext from strudel, I see both use tonejs but for tonejs to work with p5 it is built into p5 and etc. I want to try getting threejs working with strudel too

**Comment by felixroos** [comment]
I have not tried it but it would be nice to have. In case you haven't seen it yet, you can get hold of the canvas with [getDrawContext](https://github.com/tidalcycles/strudel/blob/684216bf8cd138d2e1a1c31fc7e95ca2ed776749/packages/tone/draw.mjs#L10)

I remember @guillaume-leo mentioned in discord a while ago that controlling regl or babylon would be cool too

Feel free to ask any questions / share your progress

**Comment by felixroos** [comment]
related https://github.com/tidalcycles/strudel/discussions/376
